Plugins/Public/balancemagic/Main.cpp

```diff
@@ -164,15 +177,16 @@ void LoadSettings()
 				{
 					struct DamageAdjust stEntry = { 0.0f };
 					stEntry.fighter = ini.get_value_float(0);
-					stEntry.transport = ini.get_value_float(1) ? ini.get_value_float(1) : stEntry.fighter;
-					stEntry.gunboat = ini.get_value_float(2) ? ini.get_value_float(2) : stEntry.transport;
-					stEntry.cruiser = ini.get_value_float(3) ? ini.get_value_float(3) : stEntry.gunboat;
-					stEntry.battlecruiser = ini.get_value_float(4) ? ini.get_value_float(4) : stEntry.cruiser;
-					stEntry.battleship = ini.get_value_float(5) ? ini.get_value_float(5) : stEntry.battlecruiser;
+					stEntry.freighter = ini.get_value_float(1) ? ini.get_value_float(1) : stEntry.fighter;
```

Comment:
so this will be backwards-incompatible with old inis?

Reply:
Correct. This was done at the request of the balance team. I have submitted to them an updated config based on the live one,